storybook-static/iframe.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html><html><head><meta charset="utf-8"><title>Storybook</title><meta name="viewport" content="width=device-width,initial-scale=1"><base target="_parent"><style>:not(.sb-show-main) > .sb-main,
+  :not(.sb-show-nopreview) > .sb-nopreview,
```

Comment:
Question because I'm new to this project and I opted to not include these in my PR (perhaps incorrectly): should this storybook-static directory be included in pull requests?

Reply:
Probably not.

Reply:
derp, forgot to remove these. If we're all in agreement I will also add it to the gitignore